pkg/plugins/storage/rand.go

```diff
@@ -0,0 +1,18 @@
+package storage
+
+import (
+	"math/rand"
+	"time"
+)
+
+var seededRand = rand.New(rand.NewSource(time.Now().UnixNano()))
+
+const charset = "abcdefghijklmnopqrstuvwxyz"
+
+func randomString(length int) string {
```

Comment:
this function is not used now

Reply:
BTW, if you need to do something like this later on, maybe you can use the Golang [MkdirTemp](https://pkg.go.dev/os#MkdirTemp) function instead